### PR TITLE
Make the best use of the whitespace in the UI, and center everything in the page to get the users attention.

### LINKED ISSUE #38: Make better use of whitespace in the UI
The cards would look better if they were centered rather than left aligned, and would evenly distribute the whitespace in the UI between the 2 sides. It also makes it look better IMO. Would love to know others thoughts on this.

## Screenshot
image

### PR DESCRIPTION
**Pull request by anandrajaram21**
### Why This PR Adds Value

- This PR cuts down on the final bundle size by removing unused CSS classes and making use of utility bootstrap classes, thereby keeping a consistent design language
- Before this, there was a lot of whitespace which wasn't utilized well as mentioned in issue #38. With this, the content has been centred wherever possible and the page has been mobile friendly without the need for media queries by using bootstrap

### What This PR Adds
- This PR uses the bootstrap grid system to create and centre individual items on the page, so as to grab the users attention as soon as possible.
- The bio section has been made short and concise to keep the user moving throughout the page. The user can get to know more with the social media links that will be added as mentioned in issue #3 
- This PR fixes the issue of navbar links not working as mentioned in issue #42 by adding the required IDs to the HTML attributes. 

### Screenshot

image

### How This PR Could Be Improved
- Smooth scrolling could be enabled for a smoother transition.
- For particularly long pages, a "scroll to top" button could be added.

### Issue This PR Closes

This closes #38 #40 #42 

Please do provide some feedback on how the UI design could be improved :)

### CODE REVIEW

**Review by louisefindlay23 — CHANGES_REQUESTED** [review]
@anandrajaram21, wow. You've certainly done a lot of refactoring. Great work :) Looks much nicer with centered cards.

On smaller screens though, some of the single cards look very wide because there's not much info in them. Would be nice to have a media query to account for this (reduce the card width). See screenshot below.

Portfolio Card Centering

**Comment by anandrajaram21** [comment]
Ah okay, will do. I'll open up a new issue and work on it. Thanks for telling me about it.

File: _includes/about.html [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-            <p class="number"><b>Contact:</b> {{item.number}}</p>
-            <p class="email"><b>E-mail:</b> {{item.email}}</p>
-            <p class="funfact"><b>Fun Fact:</b> {{item.funfact}}</p>
-        </div>
-   
+<h2>About</h2>
+<div class="container">
+  <div class="card">
+    <div class="row">
+      {% for item in site.data.about%}
```
Comment (louisefindlay23) [inline _includes/about.html]: Would add a space after the t here

Comment (anandrajaram21) [reply _includes/about.html]: ah, will do that right now

**Review by louisefindlay23 — APPROVED** [review]
Tested the deploy review and it looks great, @anandrajaram21. Awesome job.

**Comment by dtemir** [comment]
@anandrajaram21 looks great!

**Review by robzwolf — CHANGES_REQUESTED** [review]
Looks clean! Can you fix the merge conflicts and re-request review from me once that's done?

**Review by robzwolf — APPROVED** [review]

**Comment by robzwolf** [comment]
Great job fixing those merge requests! I know it can be a bit tough 😅

**Pull request merged**